src/outputs/parthenon_xdmf.cpp

```diff
@@ -176,14 +196,14 @@ void genXDMF(std::string hdfFile, Mesh *pm, SimTime *tm, IndexDomain domain, int
     xdmf << StringPrintf(
         "    <Grid GridType=\"Uniform\" Name=\"%s\">\n"
         "      <Topology TopologyType=\"Polyvertex\" Dimensions=\"%d\" "
-        "NodesPerElement=\"1\">\n"
+        "NodesPerElement=\"%d\">\n"
```

Comment:
That's a bugfix unrelated to this PR, isn't it?

Reply:
Yes and also the particles *still* don't work. I gave it a shot to try and fix them in this PR but failed. Thought I removed all the things I tried but I guess I missed one. This change was probably completely extraneous.